src/perception/tag_detector/tag_detector.processing.cpp

```diff
@@ -67,6 +67,9 @@ namespace mrover {
         });
         mProfiler.addEpoch("Convert");
 
+        // Call thresholding
+        publish_thresh(mImg);
```

Comment:
Rename to `publishThresholdedImage` and only do if we have listeners. We want to avoid doing any computation if we don't have to. You can use: `mThreshPub.getNumSubscribers()`